package.json

```diff
@@ -1,65 +1,99 @@
 {
-  "private": true,
-  "name": "jsxtreme-markdown",
-  "license": "MIT",
-  "scripts": {
-    "start-repl": "webpack-dev-server --config repl/webpack.config.js --content-base repl",
-    "build-repl": "PRODUCTION=true webpack --config repl/webpack.config.js",
-    "lint": "eslint .",
-    "test-jest": "jest",
-    "format": "prettier --write '**/*.js'",
-    "postinstall": "lerna bootstrap",
-    "precommit": "lint-staged",
-    "pretest": "npm run lint",
-    "test": "jest"
-  },
-  "devDependencies": {
-    "@mapbox/assembly": "^0.21.2",
-    "babel-loader": "^7.1.4",
-    "babel-preset-env": "^1.6.1",
-    "babel-preset-react": "^6.24.1",
-    "eslint": "^4.19.1",
-    "eslint-plugin-node": "^6.0.1",
-    "husky": "^0.14.3",
-    "jest": "^23.1.0",
-    "lerna": "^2.9.0",
-    "lint-staged": "^7.0.4",
-    "prettier": "1.13.7",
-    "webpack": "^3.4.1",
-    "webpack-dev-server": "^2.6.1",
-    "webpack-parallel-uglify-plugin": "^1.1.0"
-  },
-  "engines": {
-    "node": ">=4"
-  },
-  "jest": {
-    "coverageReporters": [
-      "text",
-      "html"
-    ],
-    "roots": [
-      "<rootDir>/packages"
-    ],
-    "testMatch": [
-      "**/*.test.js"
-    ],
-    "clearMocks": true,
-    "testEnvironment": "node"
-  },
-  "prettier": {
-    "singleQuote": true
-  },
-  "lint-staged": {
-    "*.js": [
-      "eslint",
-      "prettier --write",
-      "git add"
-    ]
-  },
-  "babel": {
-    "presets": [
-      "env",
-      "react"
-    ]
-  }
+	"private": true,
+	"name": "jsxtreme-markdown",
+	"license": "MIT",
+	"scripts": {
+		"start-repl": "webpack-dev-server --config repl/webpack.config.js --content-base repl",
+		"build-repl": "PRODUCTION=true webpack --config repl/webpack.config.js",
+		"lint": "eslint .",
+		"test-jest": "jest",
+		"format": "prettier --write '**/*.js'",
+		"postinstall": "lerna bootstrap",
+		"precommit": "lint-staged",
```

Comment:
```suggestion

```

With the update to lint-staged and husky, I think we can remove this line and then add a husky config to package.json:


```
"husky": {
    "hooks": {
      "pre-commit": "lint-staged"
    }
  }
```

And it may be worth running `npm run format` (perhaps, you did!), I'm not sure the pre-commit hook was functioning after it was updated.